### DISCUSSION TITLE
[Security] Bump redis to 4.5.4

### DISCUSSION THREAD

**Discussion by icecream-monster** [celery/celery#8181] [General]
- Hey! I noticed the redis version is `redis>=4.2.2,<4.4.0` 
- Due to the recent redis vulnerability discovered by ChatGPT: https://blog.sonatype.com/openai-data-leak-and-redis-race-condition-vulnerability-that-remains-unfixed , I was wondering if there is an ETA on bumping redis version to 4.5.4 in celery?
- [CVE-2023-28858](https://www.cve.org/CVERecord?id=CVE-2023-28858)
- [CVE-2023-28859](https://www.cve.org/CVERecord?id=CVE-2023-28859)

**Comment by auvipy** [comment]
I'm actually waiting for 4.3.7 back port release first

**Reply by icecream-monster** [reply]
The rating turned out to be a low/ medium risk only. I think they over-hyped it in the article saying how easy it is to leak sensitive data via race condition, ugh.
